extensions/amp-bind/0.1/amp-state.js

```diff
@@ -158,6 +163,8 @@ export class AmpState extends AMP.BaseElement {
     const ampdoc = this.getAmpDoc();
     return this.fetch_(ampdoc, this.element, isInit).then(json => {
       this.updateState_(json, isInit);
+    }, error => {
+      Services.actionServiceForDoc(this.element).trigger(this.element, 'error', AmpEvents.ERROR, ActionTrust.HIGH);
```

Comment:
(drive by comment, was triaging PRs): this can't be high-trust or it becomes a way to get around running `amp-bind` on page load.